Cargo.toml

```diff
@@ -26,3 +26,4 @@ members = [
 
 [dev-dependencies]
 shared = { path = "shared" }
+libc = { version = "0.2.14", default-features = false }
```

Comment:
This libc version is really old. Why isn't this just `0.2` ?

Reply:
No particular reason, I just followed what I could found online.
https://doc.rust-lang.org/1.30.0/unstable-book/language-features/lang-items.html#using-libc